typescript/Dockerfile

```
@@ -8,5 +8,5 @@ RUN npm install -g \
     @types/chai \ 
     @types/express \
     @types/node \
-    @types/mocha 
-    
+    @types/mocha \
+    @types/mongoose    
```

There are trailing whitespaces in here:
```suggestion
    @types/mongoose
```